Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyFragment.kt

```diff
@@ -86,9 +95,8 @@ class SurveyFragment : Fragment() {
         }
     }
 
-    @Suppress("UNUSED_PARAMETER")
-    private fun takeAPhoto(questionId: Int) {
-        // TODO: unsupported for now
+    private fun takeAPhoto() {
+        takePicture.launch(viewModel.getUriToSaveImage())
```

Comment:
You have to request `WRITE_EXTERNAL_STORAGE` permission on devices less than API 29 to be able to write on MediaStore

Reply:
Thank you! 
I don't want to handle asking for permission in this PR so, updated it to only add the camera survey question if the API is 29+